src/components/molecules/NavBar/NavBar.scss

```diff
@@ -177,112 +177,6 @@ $border-radius: 28px;
   max-width: none !important;
 }
 
-.nav-logos {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-}
-
-.nav-sparkle-logo,
-.nav-sparkle-logo_small {
-  position: relative;
-  margin-right: 12px;
-  &:hover {
-    width: 120px;
-  }
-  &.nav-sparkle-logo_small {
-    display: none;
-  }
-  div {
-    position: relative;
-    display: block;
-    width: 25px;
-    height: 35px;
-    background-image: url(/icons/sparkle-nav-logo.png);
-    background-size: 25px 35px;
-    &:before {
-      position: absolute;
-      content: "";
-      opacity: 0;
-      pointer-events: none;
-      left: 22px;
-      width: 108px;
-      height: 35px;
-      background-image: url(/icons/sparkle-nav-logo-text.png);
-      background-size: 108px 35px;
-      transform: translateX(-40px) scale(0.75);
-      transition: opacity 200ms, transform 400ms cubic-bezier(0.23, 1, 0.32, 1);
-    }
-    @keyframes sparklelogoanimate {
-      100% {
-        background-position: -75px;
-      }
-    }
-
-    &:hover {
-      opacity: 1;
-      background-image: url(/icons/sparkle-nav-logo-animate.png);
-      background-size: 75px 35px;
-      background-position: 0 0;
-      animation: sparklelogoanimate 0.4s steps(3) infinite;
-      &:before {
-        opacity: 1;
-        pointer-events: all;
-        transform: translateX(0px) scale(1);
-      }
-    }
-    &:active {
-      opacity: 0.7;
-    }
-  }
-}
-.nav-party-logo {
```

Comment:
Where did these styles go?

Reply:
NavbarSchedule. It was the only place where they are used but the level whey were defined in was incorrect. That's why I moved it.

Reply:
Cool, makes sense